sdk/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items-mlflow.ipynb

```diff
@@ -629,6 +629,64 @@
     "hd_job"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4.3. Automatic hyperparameter sweeping for your AutoML models for computer vision tasks (Automode)\n",
+    "\n",
+    "When using AutoML for Images, we can perform an automatic hyperparameter sweep to find the optimal model (Automode feature). The feature will choose a model architecture and values for the learning_rate, number_of_epochs, layers_to_freeze, etc. based on the number of runs. There is no need to specify the hyperparameter space, sampling method or early termination policy.\n",
```

Comment:
nit: "...hyperparameter sweep (automode) to find the optimal model. Automode will choose..."

Reply:
No longer calling it feature.